recipes/datamatrix/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "datamatrix" %}
+{% set pypiname = "python-" + name %}
+{% set version = "0.10.15" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: "https://pypi.io/packages/source/{{ pypiname[0] }}/{{ pypiname }}/{{ pypiname }}-{{ version }}.tar.gz"
+  sha256: "91130cd96cd8c8b7852060cccc914881f848a36f488c8d3fbf58eac47386e60a"
+
+build:
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+    - openpyxl
+    - fastnumbers
+    - prettytable
+
+test:
+  imports:
+    - datamatrix
+    - datamatrix._datamatrix
+    - datamatrix.io
+    - datamatrix.convert
+    - datamatrix.operations
+    - datamatrix.series
+    - datamatrix.functional
+  requires:
+    - numpy
+    - scipy
```

Comment:
This is odd. Why are these not runtime requirements?

Reply:
That is odd indeed. I used the recipe of the package maintainer as a reference for this one. 
As far as I remember, numpy and scipy are not strict requirements for this package to function. They do make everything faster when available. 
datamatrix is written in such a way that it can work with pure python too (as a fallback?) if c-(of rather fortran-) based dependencies such as numpy and scipy are not available. This was done at the time that this package was meant to run on an android distribution of python (on which these non-pure python dependencies were difficult to install), but I don't know if that is relevant any longer. @smathot, can you possibly shed some more light on this?

Reply:
I looked it up in the documentation of datamatrix and I appear to be correct: https://datamatrix.cogsci.nl/0.10/install/. Do you consider this point to be resolved with this information?

Reply:
@dschreij is correct. These are not strictly runtime requirements because DataMatrix can run using only the Python standard library.